util/util.go

```diff
@@ -0,0 +1,36 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+)
+
+// SplitCmd split full command to namespace, command, version.
+// ex(ns/cmd/1.0.1 => ns cmd 1.0.1)
+func SplitCmd(cmd string) (namespace, command, version string, err error) {
+	splitNamespce := strings.Split(cmd, "/")
+	if len(splitNamespce) < 2 {
+		err = fmt.Errorf("Command format is not valid")
+		return
+	}
+	splitCmdAndVer := strings.Split(splitNamespce[1], "@")
+	if len(splitCmdAndVer) < 2 {
+		err = fmt.Errorf("Command format is not valid")
+		return
+	}
+	namespace = splitNamespce[0]
+	command = splitCmdAndVer[0]
+	version = splitCmdAndVer[1]
+	return
+}
+
+// SplitCmdWithSearch search full command. If there is valid full command, return splited full command name.
+func SplitCmdWithSearch(cmds []string) (namespace, command, version string, itr int, err error) {
```

Comment:
What do you mean by "WithSearch"?

Reply:
The place of full command is not stable. So I think it will be helpful to make function which search full command from arguments and return split of full command. So  I add WithSearch.

Reply:
what is full command?

Reply:
`full command` mean the parameter of `namespace/command/version`. 
By [Screwdriver model](https://github.com/screwdriver-cd/models/pull/209/files#diff-04c6e90faac2675aa89e2176d2eec7d8R1178) they call this parameter as fullCommandName, thus I used the word `full command`.  

It is better to use formal name and sorry for use Informal name. From now I use `full command name`.

`The place` mean the place in arguments. Now we have two way to exec binary
```bash
# first way
$ sd-cmd exec namespace/command/version arguments..
# second way
$ sd-cmd namespace/command/version argumetns...
```
In first way the place of full command name is third place, but the second way full command name is second place.  
So `The place` mean the place in arguments.